src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -230,6 +230,13 @@ private void setParameterMap(
         props,
         false /* enforceDefault */);
 
+    this.checkAndUpdate(
```

Comment:
@sfc-gh-tzhang do you know why in the past we have the parameter here, but the customer can not set it? Is there any explicit reason or just missing implementation? Thanks!